opensrp-chw/src/main/java/org/smartregister/chw/util/RepositoryUtils.java

```diff
@@ -50,4 +50,6 @@ static void addDetailsColumnToFamilySearchTable(SQLiteDatabase db) {
         }
     }
 
+    String EC_REFERRAL_ADD_FP_METHOD_COLUMN = "ALTER TABLE 'ec_referral' ADD COLUMN fp_method_accepted_referral TEXT NULL;";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5646543)